service/client/client.js

```diff
@@ -110,8 +110,8 @@ export default class Client {
         return `${this.url}${this.urlVersion}/users`;
     }
 
-    getFilesRoute(teamId) {
-        return `${this.url}${this.urlVersion}/teams/${teamId}/files`;
+    getFilesRoute() {
```

Comment:
Yeah, working on the tests right now. I'll add in the route and make the changes if need be for the tests.